packages/jwt-verifier/test/integration-test/resources/testRunner.yml

```diff
@@ -20,6 +20,10 @@ scenarios:
     ports:
       applicationPort: 8080
       mockPort: 9090
+      mockHttpsPort: 9999
     command: node
+    env:
+      ISSUER: https://localhost:9999/oauth2/default
```

Comment:
My default these ports are random, more work would would need to happen to allow the ports to be injected into the configuration maybe something like `ISSUER: https://localhost:${mockHttpsPort}/oauth2/default`. But since we are setting the value directly in `mockHttpsPort` we also set it in the `ISSUER` so they stay in sync.

I'm not sure i'm following the `ISSUER` comment though

Reply:
Yes, we can make this the default value. But specifying it here is useful for code readability and also for debugging.
You can open up the well-known endpoint or simply ping `https://localhost:9999` to check if the mock auth server is running. 
Since the user can specify the `mockHttps` port in the yml config, this might change if user wants to use any other port than `9999`.
@bdemers - Any other reason we don't just default to this in TCK?

Also, I don't think we can run these TCK based ITs against your own org. The tests expect a specific client_id/client_secret and return specific auth codes back. @bdemers - correct me if I'm wrong.

Reply:
I was able to update and/or omit the `ISSUER` for the `jwt-verifier` package the tests will succeed.

Because of this, I'm assuming the tck will default to a value in certain scenarios.

> Note: Tests **do** fail when changing the `oidc-middleware` package.

Since we're not granting developers the ability to run integration tests against their own org - we should move the configuration metadata outside of these repos. Can we create a JIRA/GH Issue to change this in the future?

Reply:
Have created a JIRA to track this - https://oktainc.atlassian.net/browse/OKTA-192236

Reply:
I'm not sure i'm following. Any developer should be able to run the ITs, this config defiantly belongs in this repo.

Re the jwt-verifier, that might only be testing a happy path, not sure, but the other ITs (anything with negative tests outside the happy path would require the mock server)

Reply:
@vijetmahabaleshwar-okta the TCK defaults to random ports (so you can run tests in parallel, and not effect your development environment).

We can change this if needed in the future, but i think it is out of scope of this issue

Reply:
@bdemers 
> I'm not sure i'm following. Any developer should be able to run the ITs, this config defiantly belongs in this repo.

Developers can run the integration tests against the mock server, however they cannot run these against their Okta org. Until we provide that capability, this configuration is just noise for devs - or (*worse*) developers may attempt to change it expecting tests to work properly.

I'd propose we remove any/all configuration from this PR that isn't required for tests to run. Since the update, this only includes:

 ```
# jwt-verifier
ISSUER
```

As a future PR, we should explore pulling out the configuration or make it possible for devs to run the integration tests against the set of config values.

Reply:
I'm not sure we are talking about the same thing. Lets try to jump on a call, and i'll try to recap it here

Reply:
Per convo - lets add a comment in this file noting that we do not encourage updating this file. And will **not work** with a custom issuer URL.

Reply:
Follow up: We were talking about different things, this TCK does NOT function as a proxy (or record/play back).  We do have that type of testing in other projects historically.  This TCK (non-e2e tests) can be run without an external connection all calls to Okta are mocked (you should be able to run it on a plane)